docs/guides/web-api-basics.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: default
+title: "API Client Basics"
+lang: en
+---
+
+# API Client Basics
+
+**slack-api-client** contains simple, easy-to-use, and flexibly configurable HTTP clients for making requests to Slack APIs.
+
+Before trying the samples on this page, you need to set up your Java project first. If you haven't done it yet, check the [API Client Installation]({{ site.url | append: site.baseurl }}/guides/web-api-client-setup) guide and follow the instructions there first.
+
+## Initialize Slack Facade
```

Comment:
is Fascade a Java-specific term?

Reply:
Yes, the Facade pattern is still well-known in the Java community. But we rarely use the term recently. So, I've replaced the headline with `Initialize Slack Object` as `Initialize Slack` is a bit unclear.